src/tools/miri/tests/pass/intrinsics/portable-simd.rs

```diff
@@ -300,27 +300,6 @@ fn simd_mask() {
         }
     }
 
-    // This used to cause an ICE. It exercises simd_select_bitmask with an array as input.
-    let bitmask = u8x4::from_array([0b00001101, 0, 0, 0]);
-    assert_eq!(
-        mask32x4::from_bitmask_vector(bitmask),
-        mask32x4::from_array([true, false, true, true]),
-    );
-    let bitmask = u8x8::from_array([0b01000101, 0, 0, 0, 0, 0, 0, 0]);
-    assert_eq!(
-        mask32x8::from_bitmask_vector(bitmask),
-        mask32x8::from_array([true, false, true, false, false, false, true, false]),
-    );
-    let bitmask =
-        u8x16::from_array([0b01000101, 0b11110000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
-    assert_eq!(
-        mask32x16::from_bitmask_vector(bitmask),
-        mask32x16::from_array([
-            true, false, true, false, false, false, true, false, false, false, false, false, true,
-            true, true, true,
-        ]),
-    );
```

Comment:
As far as I can tell, this just tests `from_bitmask_vector`, so yeah if that API is gone we can remove the test. We still also directly test the intrinsic.

Does this mean simd_select_bitmask on an array is no longer used? What about simd_bitmask with an array?

Reply:
Neither intrinsic is used with arrays anymore. The limitation was const generics on the API so I could see it coming back at some point in the future, though